.github/workflows/solana.yml

```diff
@@ -15,7 +15,29 @@ defaults:
     working-directory: ./chains/solana
 
 jobs:
+  changes:
```

Comment:
As far as I could see, Solana CI is not required so we can skip without causing issues